.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,90 @@
+name: Publish Python 🐍 distribution 📦 to PyPI
+
+on: push
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.10.12"
+    - name: Install specific setuptools and pip
+      run: |
+        python -m pip install setuptools==65.5.0 pip==21
```

Comment:
Do you not need to change the wheels version here?

Reply:
The library installs just fine without `wheels` downgrade, I actually wanted to know why we change it in the first place. I know it used to take out some annoying warnings, but the last time I tested installing the library in a fresh environment, everything installed fine without any warnings.

Reply:
Interesting. We will have to talk more about this. I didn't know this was no longer necessary